TGS.Server/Instance/Repository.cs

```diff
@@ -390,8 +390,6 @@ public string Setup(string RepoURL, string BranchName)
 					if (!CompilerIdleNoLock())
 						return "Compiler is running!";
```

Comment:
Why is this an error? Compiling doesn't require access to the repo once the copy happens.

Not that it will likely ever come up.

Reply:
yeah, it's killing a fly with a machine gun but it's the cleanest method without refactoring compiling states into something other than a bool